pkg/log/ansHook_test.go

```diff
@@ -0,0 +1,331 @@
+package log
+
+import (
+	"encoding/json"
+	"github.com/SAP/jenkins-library/pkg/ans"
+	"github.com/SAP/jenkins-library/pkg/xsuaa"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"os"
+	"testing"
+	"time"
+)
+
+const testCorrelationID = "1234"
+
+func TestANSHook_Levels(t *testing.T) {
+	hook, _ := newANSHook(ans.Configuration{}, "", &ansMock{})
+	assert.Equal(t, []logrus.Level{logrus.WarnLevel, logrus.ErrorLevel, logrus.PanicLevel, logrus.FatalLevel},
+		hook.Levels())
+}
+
+func Test_newANSHook(t *testing.T) {
```

Comment:
@rodibrin I guess it doesn't make sense here either to use the list approach. I will refactor this and `TestANSHook_Fire`.

Reply:
What do you say?

Reply:
I could reduce the code substantially. But I did keep the list approach. Please review if it is better now.